### DISCUSSION TITLE
Add a toggle for displaying construction rights under the view menu

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21149] [Feature requests]
As it stands, the ability to view where your construction rights are is only possible when the land purchasing tool is opened. This means you need to have it open while building rides in order to be able to see where you can build, providing the same function that the park boundary fence provides.

If the construction rights grid was visible during actual construction, that would streamline the process of building in construction rights areas, since a scenario map layout doesn't necessarily provide visual cues for where they begin and end.

**Comment by HtotheTML** [comment]
After a bit of discussion with others, it would be more efficient to simply have it be an option to select from the view menu.

**Reply by MASHzj** [reply]
Would also help in no money scenarios.
